custom/ewsghana/models.py

```diff
@@ -101,6 +101,23 @@ def domain_object(self):
         return Domain.get_by_name(self.domain)
 
 
+class EWSMigrationStats(models.Model):
+    products_count = models.IntegerField(default=0)
```

Comment:
can we add a date field to this table that gets updated each time we run the balance step just so we can know the date and time that those numbers are valid as of